recipes/dd-handler.rb

```diff
@@ -60,7 +60,8 @@
   :use_ec2_instance_id => node['datadog']['use_ec2_instance_id'],
   :tag_prefix => node['datadog']['tag_prefix'],
   :url => node['datadog']['url'],
-  :extra_endpoints => extra_endpoints
+  :extra_endpoints => extra_endpoints,
+  :tags_blacklist_regex => ['datadog']['tags_blacklist_regex']
```

Comment:
`['datadog']['tags_blacklist_regex']` should read `node['datadog']['tags_blacklist_regex']` instead